modules/sqlite3/metadata.json

```diff
@@ -7,6 +7,9 @@
             "name": "John Millikin"
         }
     ],
+    "repository": [
+        "github:sqlite/sqlite"
```

Comment:
This is the official GitHub mirror. SQLite is developed in a [custom VCS](https://fossil-scm.org/home/doc/trunk/www/index.wiki) so I thought a Git repository link would be more useful